plugins/woocommerce-admin/client/marketplace/components/footer/footer.tsx

```diff
@@ -23,18 +23,18 @@ const refundPolicyTitle = createInterpolateElement(
 );
 
 const supportTitle = createInterpolateElement(
-	__( '<a>Support</a> teams across the world', 'woocommerce' ),
+	__( '<a>Get help</a> when you need it', 'woocommerce' ),
 	{
 		// eslint-disable-next-line jsx-a11y/anchor-has-content
 		a: <a href={ MARKETPLACE_URL + '/docs/' } />,
 	}
 );
 
 const paymentTitle = createInterpolateElement(
-	__( '<a>Safe & Secure</a> online payment', 'woocommerce' ),
+	__( '<a>Products</a> you can trust', 'woocommerce' ),
 	{
 		// eslint-disable-next-line jsx-a11y/anchor-has-content
-		a: <a href={ MARKETPLACE_URL + '/woocommerce-payments/' } />,
+		a: <a href={ MARKETPLACE_URL + '/products/' } />,
```

Comment:
☝️ I've assumed this is the intended URL for this link.

Reply:
Yeah, I'd say so.